tests/tests/hafs_regional_storm_following_1nest_atm_ocn_debug

```diff
@@ -0,0 +1,136 @@
+###############################################################################
+#
+#  HAFS regional with 1 storm-following moving nest and atmosphere-ocean coupled with HYCOM
+#
+###############################################################################
+
+export TEST_DESCR="Compare HAFS regional with 1 storm-following moving nest and atmosphere-ocean coupled results with previous trunk version"
+
+export CNTL_DIR=hafs_regional_storm_following_1nest_atm_ocn
```

Comment:
The control directory should be the debug test.

Reply:
Oops, you are right! Sorry, I missed that. @jiandewang would you please change this to hafs_regional_storm_following_1nest_atm_ocn_debug

Reply:
I think we have a problem w/ the baseline. When this test ran, during the RT, it would have created a debug baseline but called it ``hafs_regional_storm_following_1nest_atm_ocn``. Now how did the non-debug version of that pass against that?

Reply:
I think we have to re-run hafs_regional_storm_following_1nest_atm_ocn and hafs_regional_storm_following_1nest_atm_ocn_debug to confirm.

Reply:
strange! hafs_regional_storm_following_1nest_atm_ocn_debug run has some trouble to copy over input file (grid_spec,nc).

Reply:
grid_spec is being copied from a directory name that includes the test name, for example:

input-data-20221101/FV3_hafs_input_data/INPUT_hafs_regional_storm_following_1nest_atm/grid_spec.nc

Reply:
@jkbk2004 yes, I only ran debug.